decidim-assemblies/lib/decidim/assemblies/admin_engine.rb

```diff
@@ -14,14 +14,14 @@ class AdminEngine < ::Rails::Engine
       paths["db/migrate"] = nil
 
       routes do
-        resources :assemblies do
+        resources :assemblies, param: :slug, except: :show do
```

Comment:
The param name `slug` why sometimes is called like this and sometimes like `assembly_slug`? Is it intended?

Reply:
I simply renamed the old `:id`/`:assembly_id` to use the slug. I guess `:assembly_id` was used to differentiate assembly IDs from feature IDs. I don't want to change this because we might move features to use slugs in the URLs, so this is good.